packages/angular_devkit/schematics/tools/file-system-engine-host-base.ts

```diff
@@ -91,16 +103,16 @@ export class SchematicNameCollisionException extends BaseException {
  * all other EngineHost provided by the tooling part of the Schematics library.
  */
 export abstract class FileSystemEngineHostBase implements
-    EngineHost<FileSystemCollectionDescription, FileSystemSchematicDescription> {
+  EngineHost<FileSystemCollectionDescription, FileSystemSchematicDescription> {
```

Comment:
Why all the style changes? Your PR should only include code you changed.

Reply:
Yeah agreeded  have done Autoformat when fixing the rebase conflict. 

Will fix this later